cardinal-components-block/src/main/java/dev/onyxstudios/cca/internal/CardinalComponentsBlock.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Cardinal-Components-API
+ * Copyright (C) 2019-2020 OnyxStudios
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+ * OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE
+ * OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+package dev.onyxstudios.cca.internal;
+
+import dev.onyxstudios.cca.api.v3.block.BlockEntitySyncedComponent;
+import dev.onyxstudios.cca.internal.base.ComponentsInternals;
+import nerdhub.cardinal.components.api.ComponentRegistry;
+import nerdhub.cardinal.components.api.ComponentType;
+import nerdhub.cardinal.components.api.component.extension.SyncedComponent;
+import net.fabricmc.fabric.api.network.ClientSidePacketRegistry;
+import net.fabricmc.loader.api.FabricLoader;
+import net.minecraft.network.PacketByteBuf;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.math.BlockPos;
+
+public class CardinalComponentsBlock {
+    // Safe to put in the same class as no client-only class is directly referenced
+    public static void initClient() {
```

Comment:
if this method is supposed to be client-only, maybe annotate it as such?
perhaps also suppress the 'unused' linter warning, since this method is used as static mod initializer.

Reply:
I personally do not think applying environment annotations to mod classes is a good idea unless strictly necessary. The linter warning suppression is a good idea though.